packages/fiori-generator-shared/src/environment.ts

```diff
@@ -0,0 +1,30 @@
+import { isAppStudio } from '@sap-ux/btp-utils';
+
+export const hostEnvironment = {
+    vscode: {
+        name: 'Visual Studio Code',
+        technical: 'VSCode'
+    },
+    bas: {
+        name: 'SAP Business Application Studio',
+        technical: 'SBAS'
+    },
+    cli: {
+        name: 'CLI',
+        technical: 'CLI'
+    }
+};
+
+/**
+ * Determine if the current prompting environment is cli or a hosted extension (app studio or vscode).
+ *
+ * @param isYUI - is the current environment YUI
+ * @returns the platform name and technical name
+ */
+export function getHostEnvironment(isYUI: boolean): { name: string; technical: string } {
```

Comment:
Not sure about this one. How will the consumer determine if its YUI or not. If this is to be a shared function then it needs to be isolated and determine the mhost environment without any inputs.

Reply:
I've updated `getHostEnvironment` as discussed

Reply:
Thx